lang/js/src/main/scala/com/wavesplatform/lang/WavesContext.scala

```diff
@@ -4,13 +4,14 @@ import com.wavesplatform.lang.v1.traits.{DataType, Environment, Recipient, Tx}
 
 object WavesContext {
   val env = new Environment {
-    override def height: Int                                                                   = impl.Environment.height
-    override def networkByte: Byte                                                             = impl.Environment.networkByte
-    override def transaction: Tx                                                               = impl.Environment.transaction
-    override def transactionById(id: Array[Byte]): Option[Tx]                                  = impl.Environment.transactionById(id)
-    override def transactionHeightById(id: Array[Byte]): Option[Int]                           = impl.Environment.transactionHeightById(id)
-    override def data(addressBytes: Array[Byte], key: String, dataType: DataType): Option[Any] = impl.Environment.data(addressBytes, key, dataType)
-    override def resolveAlias(name: String): Either[String, Recipient.Address]                 = impl.Environment.resolveAddress(name)
+    override def height: Int                                         = impl.Environment.height
+    override def networkByte: Byte                                   = impl.Environment.networkByte
+    override def transaction: Tx                                     = impl.Environment.transaction
+    override def transactionById(id: Array[Byte]): Option[Tx]        = impl.Environment.transactionById(id)
+    override def transactionHeightById(id: Array[Byte]): Option[Int] = impl.Environment.transactionHeightById(id)
+    override def data(addressOrAlias: Array[Byte], key: String, dataType: DataType): Option[Any] =
```

Comment:
`addressOrAlias` should be of type `Recipient`(maybe we can find better naming), not `Array[Byte]`